pkg/operator/ceph/pool/controller.go

```diff
@@ -497,3 +502,42 @@ func (r *ReconcileCephBlockPool) cancelMirrorMonitoring(cephBlockPool *cephv1.Ce
 		delete(r.blockPoolContexts, channelKey)
 	}
 }
+
+func (r *ReconcileCephBlockPool) disableMirroring(pool *cephv1.CephBlockPool) error {
+	mirrorInfo, err := cephclient.GetPoolMirroringInfo(r.context, r.clusterInfo, pool.Name)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get mirroring info for the pool %q", pool.Name)
+	}
+
+	if mirrorInfo.Mode == "image" {
+		mirroredPools, err := cephclient.GetMirroredPoolImages(r.context, r.clusterInfo, pool.Name)
+		if err != nil {
+			return errors.Wrapf(err, "failed to list mirrored images for pool %q", pool.Name)
+		}
+
+		if len(*mirroredPools.Images) > 0 {
+			logger.Warningf("manually disable mirroring on images in the pool %q", pool.Name)
+			return nil
```

Comment:
```suggestion
if len(*mirroredPools.Images) > 0 {
	msg:=fmt.Sprintf("there are images in  %q, please disable mirroring for each images", pool.Name)
	logger.Errorf(msg)
	return errors.New(msg)
}
```

see above suggestion makes sense as we need to wait for user to disable mirroring on all the images first